cmd/analyzer/subcmds/vpcconfigs.go

```diff
@@ -76,30 +50,15 @@ func parseProviderFromFile(fileName string) (string, error) {
 }
 
 func vpcConfigsFromFiles(fileNames []string, inArgs *inArgs) (*vpcmodel.MultipleVPCConfigs, error) {
-	var mergedRC *datamodel.ResourcesContainerModel
-	for _, file := range fileNames {
-		provider, err := parseProviderFromFile(file)
-		if err != nil {
-			return nil, err
-		}
-		if provider == common.IBM {
-			rc, err1 := ibmvpc.ParseResourcesFromFile(file)
-			if err1 != nil {
-				return nil, fmt.Errorf("error parsing input vpc resources file: %w", err1)
-			}
-			mergedRC, err1 = mergeResourcesContainers(mergedRC, rc)
-			if err1 != nil {
-				return nil, err1
-			}
-		} else {
-			return nil, fmt.Errorf(notSupportedYet, provider)
-		}
+	// len(fileNames) can not be zero
+	provider, err := parseProviderFromFile(fileNames[0])
```

Comment:
what is the error message in case the provider is not supported?

Reply:
will add this in vpcConfigsFromFiles func after switch case